Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -650,10 +696,14 @@ function ViewPanelHousing( data:table )
     --Improvements
     CQUI_BuildHousingBubbleInstance("ICON_CITYSTATE_INDUSTRIAL", CQUI_HousingFromImprovements, "LOC_PEDIA_IMPROVEMENTS_PAGEGROUP_IMPROVEMENTS_NAME");    -- CQUI real housing from improvements value
     --Era
-    CQUI_BuildHousingBubbleInstance("ICON_GREAT_PERSON_CLASS_SCIENTIST", data.HousingFromStartingEra, "LOC_GAME_START_ERA");
+    data.HousingFromStartingEra = data.HousingFromStartingEra or 0;
```

Comment:
the unmodified Firaxis version of this file only shows Housing from Era if the value is not zero